### DISCUSSION TITLE
better ui

### DISCUSSION THREAD

**Discussion by 1gzify** [Paymenter/Paymenter#237] [Ideas]
Make a better ui or just create a new free theme for us. Im not really a fan of the current theme. (not colors)

**Comment by ProgyNL** [comment]
https://paymenter.org/docs/customization/create-a-theme/

**Reply by 1gzify** [reply]
no like paymenter itself updates the theme the current one is not good at all

**Comment by 09han09** [comment]
Make a design or come with suggestions on what to change please.

**Reply by 1gzify** [reply]
The whole customer page ui. Not the admin.

**Comment by 09han09** [comment]
Yeah then come with suggestions on how to improve it.
